src/components/Alphabet/Letter/Letter.css

```diff
@@ -16,4 +16,20 @@
     @media (min-width: 768px) {
         font-size: 35px;
     }
+}
+
+.letterHover:hover {
+    color: #191919;
+    background-color: #dedede;
+    border-color: #000;
+}
+
+.correctLetter {
+    background-color: #157c00;
+    cursor: default;
```

Comment:
Instead, you can add pointer-event: none, which will block hover and other mouse effects (it's more common)
But that's up to you

Reply:
Ok, I used yours